packages/warriorjs-abilities/src/attack.js

```diff
@@ -3,15 +3,17 @@ import { BACKWARD, FORWARD } from '@warriorjs/geography';
 const defaultDirection = FORWARD;
 
 function attack({ power }) {
+  const backwardPower = Math.ceil(power / 2.0);
   return unit => ({
     action: true,
-    description: `Attacks a unit in the given direction (\`'${defaultDirection}'\` by default), dealing ${power} HP of damage.`,
+    argumentsDescription: `direction = '${defaultDirection}'`,
+    description: `Attacks a unit in the given direction (${defaultDirection} by default), dealing ${power} HP of damage in any direction except backward (reduced to ${backwardPower} if attacking backward).`,
```

Comment:
I like the addition you did! I find the wording unnecessarily long though. What about:

```suggestion
    description: `Attacks a unit in the given direction (${defaultDirection} by default), dealing ${power} HP of damage (${backwardPower} HP if attacking backward).`,
```